src/main/java/org/tuberlin/de/best_taxidriver/BestTaxidriver.java

```diff
@@ -0,0 +1,138 @@
+package org.tuberlin.de.best_taxidriver;
+
+import org.apache.flink.api.common.functions.*;
+import org.apache.flink.api.common.operators.Order;
+import org.apache.flink.api.java.DataSet;
+import org.apache.flink.api.java.ExecutionEnvironment;
+
+import org.apache.flink.api.java.tuple.Tuple11;
+import org.tuberlin.de.read_data.Taxidrive;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.RandomAccessFile;
+
+
+/**
+ * Created by gerrit on 23.10.15.
+ */
+public final class BestTaxidriver {
+
+    public static void main(String[] args) throws Exception {
+        // set up the execution environment
+        final ExecutionEnvironment env = ExecutionEnvironment.getExecutionEnvironment();
+
+        // load taxi data from csv-file
+        DataSet<Taxidrive> taxidrives = env.readCsvFile("data/sorted_data.csv")
```

Comment:
I agree. BestTaxiDriver class is still under development, so i try to remove it from pull request.